cytoflow/tests/test_channel_stat.py

```diff
@@ -74,9 +73,11 @@ def testSubset(self):
                                      subset = "Dox == 10.0",
                                      function = len).apply(self.ex)
         stat = ex.statistics[("ByDox", "len")]
-       
-        self.assertEqual(stat.loc[False], 24801)
```

Comment:
Verified that these are incorrect by direct comparison to a `groupby` in pandas; I've added that code to the PR.
```
T      Dox
False  1.0     9953
       10.0    5601
True   1.0       47
       10.0    4399
dtype: int64
```

Note that when I run these new tests, they pass.

Reply:
These change a little bit each time I upgrade `statsmodels`. Probably should use some fuzzier matching...